tests/flow/destructuring/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -458,27 +458,23 @@ function tup_pattern2<X>([proj]: Proj<X>) {} // proj: X
 function rest_antipattern<T>(...t: T) {} // nonsense
 function rest_pattern<X>(...r: X[]) {} // r: X[]
 
-function obj_rest_pattern<X>(
-  { _, ...o }: { _: any, x: X } // o: { x: X }
-) {
+function obj_rest_pattern<X>({ _, ...o }: { _: any, x: X }) {
+  // o: { x: X }
```

Comment:
The new behavior is more correct. In order to add a comment on the `{`, we put it on the next line.